### PR TITLE
New package: stgit 1.5

### PR DESCRIPTION
**Pull request by chexum**
stgit (command stg) is a git patch management tool that helps manage multiple pending patches without banishing them to a separate branch.

#### Testing the changes
- I tested the changes in this PR: **YES** (daily)

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/stgit/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+pkgname=stgit
+version=1.5
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools git"
+short_desc="Git patch management to keep commits as stack of patches"
+maintainer="J Farkas <chexum+git@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://stacked-git.github.io/"
+changelog="https://stacked-git.github.io/changelog/"
```
Comment (meator) [inline srcpkgs/stgit/template:11]: Raw text files are preferred for `changelog`s.
```suggestion
changelog="https://raw.githubusercontent.com/stacked-git/stgit/master/CHANGELOG.md"
```

**Comment by chexum** [comment]
Thanks @meator (github didn't let me type into the comment field for resolution).  Amended/rebased/pushed.

File: srcpkgs/stgit/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'stgit'
+pkgname=stgit
+version=1.5
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools git"
```
Comment (Chocimier) [inline srcpkgs/stgit/template:6]: Doesn't need git build time, but needs git and python3 runtime. Please add `depends`.

**Comment by chexum** [comment]
Thank you for the comments, I'll try to get around to reviewing in substance soon

**Comment by chexum** [comment]
still working on getting back on my feet and retesting this package, appreciate the patience

**Pull request closed without merging**